### ISSUE TITLE
Problem about upload my env to bioconda  

### ISSUE THREAD

**Issue by JeremyQuo** [hasindu2008/slow5lib#71]
when I add 'pyslow5' in my setup.py
```
    install_requires=[
        'pyslow5'
    ],

```
And want to generate env file by `conda skeleton pypi`
And it returned,

No eggs found in /tmp/easy_install-w8j87miq/pyslow5-1.0.0/egg-dist-tmp-8su1pjdd (setup script problem?)
error: The 'pyslow5==1.0.0' distribution was not found and is required by xxxx

**Comment by Psy-Fer** [comment]
Hey,

I see you closed the issue. Did you figure out why this happens? Could you let me know in case it happens to someone else?

Cheers,
James.

**Comment by JeremyQuo** [comment]
Thanks for your interest.
Actually, I did not solve this problem and played a trick.
For my Pypi package, I release two versions including v0.0.1 with the pyslow5 requirement and another(v0.0.2) without
So the requirement about pyslow5 in setup.py (v0.0.1) will be deleted to generate the yaml file.
Afterward, I will manually modify the YAML file by updating the version number to v0.0.2  as well as the sha256 value by hand.
I don't know whether it will work, biconda is still checking and I will update the answer if I applied the additional method.
